src/it/scala/inox/tip/TipParsingSuite.scala

```diff
@@ -0,0 +1,13 @@
+/* Copyright 2009-2016 EPFL, Lausanne */
+
+package inox
+package tip
+
+class TipParsingSuite extends TestSuite with ResourceUtils {
+  for (file <- resourceFiles("regression/tip/PARSING", _.endsWith(".tip"))) {
+    test(s"PARSING - ${file.getName}") { ctx =>
+      new Parser(file).parseScript
```

Comment:
I think it would be best to also check whether the parsed script typechecks.
You can plug into the TipPrintingSuite with the PARSING sources, that way you'll get a more comprehensive test. Also, you should maybe change the TipPrintingSuite to call `ensureWellFormed` instead of the ad-hoc type-checking code currently used.